Angular/src/app/messenger-page/right-box/sender-bar/sender-bar.component.ts

```diff
@@ -1,15 +1,32 @@
 import { Component, OnInit } from '@angular/core';
+import { Globals } from 'src/app/shared/globals';
+import { ApiService } from 'src/app/services/api.service';
+import { ChatService } from 'src/app/services/chat.service';
+import { ConversationModel } from 'src/app/models/conversation.model';
 
 @Component({
   selector: 'app-sender-bar',
   templateUrl: './sender-bar.component.html',
   styleUrls: ['./sender-bar.component.css']
 })
 export class SenderBarComponent implements OnInit {
+  constructor(private chatService: ChatService, private apiService: ApiService) { }
 
-  constructor() { }
+  private conversation: ConversationModel;
 
   ngOnInit() {
+    this.chatService.currentConversation
+    .subscribe((data) => this.conversation = data);
   }
 
+  sendMessage(messageInput: any) {
+    if (messageInput.value === '') { // TODO do not allow whitespace strings - regex?
```

Comment:
first trim and then compare to empty string
https://www.w3schools.com/jsref/jsref_trim_string.asp

Reply:
Good catch :D  I was searching for any function called strip (python), something like that,  and it's apparently called differently in js.